helm/gateway-api-config/values.yaml

```diff
@@ -2,3 +2,7 @@ gateway:
   name: default
   class: default
   host: test.gaws.gigantic.io
+
+gatewayClass:
+  - name: eg
```

Comment:
why `eg`?

Reply:
it's what envoy-gateway uses in their tutorials. We could name it giantswarm-default or something like that. wdyt?

Reply:
I like `giantswarm-default`

Reply:
```suggestion
  - name: giantswarm-default
```